yaksh/views.py

```diff
@@ -420,11 +420,11 @@ def start(request, questionpaper_id=None, attempt_num=None):
         if is_moderator(user):
             return redirect("/exam/manage")
         return redirect("/exam/quizzes")
+    if not last_attempt:
+        attempt_number = 1
+    else:
+        last_attempt.attempt_number + 1
```

Comment:
This statement looks to be meaningless as it does not explicitly set anything, did you mean:
`attempt_number = last_attempt.attempt_number + 1`?  Why wasn't this caught by the test?

Reply:
Resolved.